.rubocop.yml

```diff
@@ -38,5 +38,7 @@ RSpec/DescribeClass:
     - tasks/**/*.rb
 RSpec/ExampleLength:
   Enabled: false
+RSpec/NestedGroups:
+  Max: 4
```

Comment:
Will there not be lots of violations in the existing codebase?

Reply:
There were a few in the tests I touched, and it was slightly easier to increase the allowed nesting by one.
If you think it's better leave it to 3, I'll go back and try to un-nest the specs instead, but my opinion is that it can be left for a later time